src/Microsoft.DotNet.Tools.Compiler.Native/Program.cs

```diff
@@ -99,8 +31,8 @@ private static int ExecuteApp(string[] args)
 
             try
             {
-                var cmdLineArgs = GetArgs(args);
-                var config = ParseAndValidateArgs(cmdLineArgs);
+                var cmdLineArgs = ArgumentsParser.Parse(args);
+                var config = cmdLineArgs.GetNativeCompileSettings();
```

Comment:
The class ArgValues is mostly being used as a bridge between the NativeCompileSettings and ArgumentsParser it seems. Would it make sense to return NativeCompileSettings directly from the ArgumentsParser?